snippets/nordic-bt-rpc/boards/nrf54h20dk_nrf54h20-mem-map-move.dtsi

```diff
@@ -18,6 +18,7 @@
 &mram1x {
 	cpurad_rw_partitions: cpurad-rw-partitions {
 		compatible = "nordic,owned-partitions", "fixed-partitions";
+		status = "disabled";
```

Comment:
This makes the snippet unusable in case there are SETTINGS used by a BLE protocol. It doesn't look like proper fix for a problem

Reply:
The partition is enabled for the Radio core in https://github.com/nrfconnect/sdk-nrf/blob/main/snippets/nordic-bt-rpc/boards/nrf54h20dk_nrf54h20_cpurad.overlay#L12
It will be disabled only for the App core, which already has its storage partition located in an adjacent block.